src/subdomains/staking/application/services/liquidity-management.service.ts

```diff
@@ -51,9 +51,23 @@ export class LiquidityManagementService {
 
     try {
       await this.checkMasternodesInProcess();
+      const updatedList = await this.masternodeService.getAllWithStates([
+        MasternodeState.ENABLING,
+        MasternodeState.MOVING_COLLATERAL,
+      ]);
+
+      const enabling = updatedList.filter((node) => node.state === MasternodeState.ENABLING).length;
+      const movingCollateral = updatedList.filter((node) => node.state === MasternodeState.MOVING_COLLATERAL).length;
+
+      if (enabling > 0 || movingCollateral > 0) {
+        console.info(
+          `Stopping masternode cronjob due to ${enabling} enabling or ${movingCollateral} moving collateral masternodes`,
+        );
```

Comment:
Do you see a benefit in knowing how many masternodes are "dirty" right now?

Reply:
no not really, but we had it too often in the last weeks, that we would need more information. Thats why I have thought print it, if it isn't used in any meaningful way, then we can remove it